packages/mediacenter/kodi/scripts/service-addon-wrapper

```diff
@@ -64,3 +64,48 @@ if [ -f "${SERVICE_FILE}" ] ; then
       ;;
   esac
 fi
+
+if [ ! -d /storage/.cache/kernel-overlays ] ; then
+  mkdir -p /storage/.cache/kernel-overlays
+fi
+
+# kernel-overlay addons built into the image have their
+# files installed in the default /usr/lib/kernel-overlays
+# location, not inside the kodi addon dir
+
+case "${ADDON_PATH}" in
+  /usr/share/kodi/addons/*)
+    OVERLAY_PATH="/usr/lib/kernel-overlays/${ADDON_ID}"
+    ;;
+  *)
+    OVERLAY_PATH="${ADDON_PATH}/kernel-overlay"
+    ;;
+esac
+
+create_overlay_conf() {
+  rm -f "${OVERLAY_CONF}"
```

Comment:
No need for the `rm -f` as the next line will overwrite any existing file. I think I'd probably get rid of the function and just inline the echo where it is needed further down.